playbooks/install.py

```diff
@@ -1,10 +1,112 @@
-# wget setup_frappe.py | python3
-import os, sys, subprocess, getpass, json, multiprocessing, shutil, platform
-from distutils.spawn import find_executable
+#!/usr/bin/env python3
+import os, sys, subprocess, getpass, json, multiprocessing, shutil, platform, warnings
+from builtins import print as _
 
 tmp_bench_repo = '/tmp/.bench'
+tmp_log_folder = '/tmp/logs/'
+log_path = os.path.join(tmp_log_folder, 'install_bench.log')
+log_stream = sys.stdout
+
+
+def print(message, level=0):
+	levels = {
+		0: '\033[94m',	# normal
+		1: '\033[92m',	# success
+		2: '\033[91m',	# fail
+		3: '\033[93m'	# warn/suggest
+	}
+	start = levels.get(level) or ''
+	end = '\033[0m'
+	_(start + message + end)
+
+
+def find_executable(executable, path=None):
```

Comment:
why?

Reply:
why coloured outputs?

Reply:
why are we not importing `find_executable` directly?

Reply:
`distutils` has been excluded from the default python now. So that translates to installing it before running our script or just pasting the function here. https://github.com/pypa/get-pip/issues/43#issuecomment-533542269

Reply:
how about using `shutil.which()` then instead?

Reply:
works